.drone.yml

```diff
@@ -1,39 +1,62 @@
 branches:
-  exclude: [ gh-pages ]
-
+    exclude:
+        - gh-pages
 matrix:
-  TEST:
-    - audit
-    - conventions
-    - examples
+    TEST:
+        - audit
+        - conventions
+        - examples
+        - visual-regression
 
 pipeline:
-  install:
-    image: node:dubnium
-    commands:
-      - yarn install --frozen-lockfile
+    audit:
+        image: 'node:dubnium'
+        commands:
+            - 'yarn install --frozen-lockfile'
+            - 'yarn audit'
+        when:
+            matrix: {TEST: audit}
+
+    conventions:
+        image: 'node:dubnium'
+        commands:
+            - 'yarn install --frozen-lockfile'
+            - 'yarn test:coding-conventions'
+        when:
+            matrix: {TEST: conventions}
+
+    examples:
+        image: 'node:dubnium'
+        commands:
+            - 'yarn install --frozen-lockfile'
+            - 'cd examples/webpack4 && yarn build --display=errors-only'
+            - 'cd ../webpack4-eu && yarn build --display=errors-only'
+        when:
+            matrix: {TEST: examples}
 
-  audit:
-    image: node:dubnium
-    commands:
-      - yarn audit
-    when:
-      matrix:
-        TEST: audit
+    s3:
+        image: kalinchernev/node-dubnium-aws-cli
```

Comment:
I searched really hard for an image which could match our environment, at one point I setup drone locally to avoid pushing changes in this pull request. If you know an image which provide the right environment, I'll change it.

The trickiest part is this:

```
error eslint@5.16.0: The engine "node" is incompatible with this module. Expected version "^6.14.0 || ^8.10.0 || >=9.10.0".
```

And linux alpine appeared to be "strict" in versions of node it provides, etc.

So finally, not to lose time on this, I extended the node dubnium which we use in the other steps and added the aws cli, that's all.

Reply:
all the pipelines seem to run `yarn install --frozen-lockfile`

wondering why we don't have the separate `install` step as we used to have: https://github.com/ec-europa/europa-component-library/blob/v1/.drone.yml#L25

if we put it in another step, then we could reuse the same image as in v1, https://github.com/ec-europa/europa-component-library/blob/v1/.drone.yml#L72, or a more recent one

Reply:
I was wondering about this as well, but I followed the way of the [last commit](https://github.com/ec-europa/europa-component-library/blob/0d8c026de11dbb060de88a89884572fbd80be4f6/.drone.yml) which I took from. I'll change it to have a separate install step.

regarding the image, I tried `garland/aws-cli-docker` of course, but [it doesn't have yarn](https://drone.fpfis.eu/ec-europa/europa-component-library/3683/12). If you have a better idea, please let me know.

Reply:
you could build storybook in a previous step maybe, still in the same pipeline (TEST: visual-regression) 🤔